flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/scheduler/SchedulerIsolatedTasksTest.java

```diff
@@ -180,61 +119,14 @@ public void testScheduleQueueing() throws Exception {
 				testingSlotProvider.getNumberOfAvailableSlots());
 	}
 	
-	@Test
-	public void testScheduleWithDyingInstances() throws Exception {
```

Comment:
I think this scenario is still valid. Looks to me it's also testing slot releasing due to TM lost.
Can we keep this case but adjust the checking logics?
Rather than expect a `NoResourceAvailableException` to be thrown, we should expect the request to be not completed.

Reply:
I agree with @zhuzhurk. We should make sure that this functionality is either covered by another test or we need to add it.